www/front_src/src/Resources/Graph/Performance/Graph/Annotations/EventAnnotations.tsx

```diff
@@ -29,7 +29,7 @@ const EventAnnotations = ({
   color,
   resourceId,
 }: Props): JSX.Element => {
-  const events = filter<TimelineEvent>(propEq('type', type), timeline);
+  const events = filter(propEq('type', type), timeline);
```

Comment:
Using `@types/ramda v0.27.45`, the typing `<TimelineEvent>` should work

Reply:
It's not needed though as the type is properly infered without the generic